charts/kamaji-etcd/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
+version: 0.4.0
```

Comment:
A minor change is required since the `Datastore` manifest will be part of the managed resources, as the opposite for the RBAC-related one (Role, Rolebinding, ServiceAccount).